task/01-strings-tasks.js

```diff
@@ -238,7 +326,8 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+    const isString = typeof value === 'string' || value instanceof String
+    return isString
```

Comment:
We can simply write `return typeof value === 'string' || value instanceof String`, additional variable will be redundant in this case